clean_log_cron.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- hosts: pd_servers
+  tags:
+    - pd
+  become: true
```

Comment:
If adding cron doesn't nedd root privilege, you should add become: true in `restart crond` task.